src/NavigationBar/index.js

```diff
@@ -0,0 +1,214 @@
+        }}
+        onPress={actions}
+      />
+    )
+  }
+
+  const renderAdditionalItems = () => {
+    return <View />
+  }
+
+  const renderNavBar = () => {
+    let variant =
+      _width > DeviceBreakpoint.TABLET_BREAKPOINT ? 'desktop' : 'mobile'
+
+    let containerStyles = {
+      backgroundColor,
+      height: 76,
+      paddingTop: 32,
+      paddingRight: 20,
+      paddingLeft: variant === 'desktop' ? 20 : 12,
+      paddingBottom: 12,
+      flexDirection: 'row',
+      alignItems: 'center',
+      ...getBorderStyle(),
+      ...getShadowStyle(shadow),
+    }
+    if (!editor) {
+      containerStyles = {
+        ...containerStyles,
+        height: 106,
+        paddingTop: 50,
+        marginTop: -30,
+        ...getBorderStyle(),
+      }
+    }
+
+    // Logic for centering the title and menu items on mobile and desktop respectively
+    const centerStyles = {
+      position: 'absolute',
+      display: 'inline-flex',
+      justifyContent: 'center',
+      left: 0,
+      right: 0,
+      margin: '0 auto',
+    }
+
+    let menuItemsStyles = {
+      flexDirection: 'row',
+      marginLeft: menuItems.alignment === 'right' ? 'auto' : '',
+    }
+
+    if (menuItems.alignment === 'center') {
+      menuItemsStyles = {
+        ...menuItemsStyles,
+        ...centerStyles,
+      }
+    }
+
+    let mobileAlignment = title.universalLayout ? 'left' : title.mobileAlignment
+    let mobileTitleStyles = {
+      marginLeft: mobileAlignment === 'right' ? 'auto' : '',
+      paddingRight: '12px',
+      flexDirection: 'row',
+    }
+
+    if (mobileAlignment === 'center') {
+      mobileTitleStyles = {
+        ...mobileTitleStyles,
+        ...centerStyles,
+      }
+    }
+
+    if (variant === 'desktop') {
+      // render the title on the left, the menu items in the middle, and the profile image on the right
+      return (
+        <View style={containerStyles}>
+          <View style={{ paddingRight: '26px', justifyContent: 'flex-start' }}>
+            <Title variant={variant} titleOptions={title} />
+          </View>
+          <View style={menuItemsStyles}>
+            <MenuItems
+              menuItems={menuItems}
+              variant={variant}
+              activeMenuItem={activeMenuItem}
+              setActiveMenuItem={setActiveMenuItem}
+              items={items}
+              _fonts={_fonts}
+            ></MenuItems>
+          </View>
+          <View
+            style={{
+              marginLeft: menuItems.alignment !== 'right' ? 'auto' : '',
+            }}
+          >
+            {renderProfileImage()}
+            {renderAdditionalItems()}
+          </View>
+        </View>
+      )
+    } else if (variant === 'mobile') {
+      return (
+        <View style={containerStyles}>
+          <View style={mobileTitleStyles}>
+            <Title variant={variant} titleOptions={title} />
+          </View>
+          <View
+            style={{ marginLeft: mobileAlignment !== 'right' ? 'auto' : '' }}
```

Comment:
Keeping conditionals outside of jsx allows to separate the presentation and logic layer

usually a 
`const marginLeft = mobileAlignment !== 'right' ? 'auto' : ''`
before the jsx might do the trick